_data/master/navbars/security.yml

```diff
@@ -50,7 +50,11 @@ toc:
   path: /security/defend-dos-attack
 - title: Apply policy to Kubernetes node ports
   path: /security/kubernetes-node-ports
+- title: Use service accounts in policy rules
+  path: /security/service-accounts
 - title: Using HTTP methods and paths in policy rules
-  path: /security/http-methods 
+  path: /security/http-methods
 - title: Enable extreme high-connection workloads
-  path: /security/high-connection-workloads   
\ No newline at end of file
+  path: /security/high-connection-workloads   
+- title: Use service accounts in policy rules
```

Comment:
This shows up twice in the navbar (a few lines above as well)

Reply:
Fixed.

Reply:
Fixed.